app/root/root.css

```diff
@@ -938,14 +941,23 @@ code .comment {
 }
 
 .terminal-actions {
-  right: 32px;
-  top: 22px;
```

Comment:
Does not specifying the top work for non-dense mode too?

Reply:
Yep, it works due to the new `.terminal-container` wrapper div around the terminal component that has `position: relative`